plugins/inputs/socketstat/testdata.go

```diff
@@ -0,0 +1,62 @@
+package socketstat
+
+var tests = []struct {
```

Comment:
What I meant here is to generate multiple _text_ files for the test cases that can be generated via `ss -... > some_test_file` and the define the expected metrics in the tests. Take a look at e.g. [this plugin](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/nvidia_smi/nvidia_smi_test.go).

Reply:
fair enough ;) I pushed a fix to implement that.